message_ix/model/MESSAGE/model_core.gms

```diff
@@ -560,6 +561,18 @@ RESOURCE_HORIZON(node,commodity,grade)$( SUM(year$map_resource(node,commodity,gr
 *         \cdot duration\_time\_rel_{h,h^A} \cdot ACT_{n^L,t,y^V,y,m,h^A} & \\
 *     - \sum_{\substack{n^L,t,m,h^A \\ y^V \leq y}} input_{n^L,t,y^V,y,m,n,c,l,h^A,h}
 *         \cdot duration\_time\_rel_{h,h^A} \cdot ACT_{n^L,t,m,y,h^A} & \\
+*     + \sum_{\substack{n^L,t}} output\_cap\_new_{n^L,t,y,n,c,l,h}
```

Comment:
I tried to align them all to the right side but the lines with fewer entries constantly shift to the left even when I try to combine two short lines.. So if you know how to do that, please make the change. 

About the first-year formulation, in the latest version, there are now two different formulations for the first year also based on a condition of remaning_capacity. So I was not sure whether to include all those special conditions in the main formulation as it might look too crowded. Maybe it is better to describe the differences in a paragraph and direct the reader to the model_core.py file if he/she wants to see more details?